tests/test_tftools.py

```diff
@@ -94,3 +95,20 @@ def test_batch_norm_3d():
     x = tf.placeholder(tf.float32, input_shape)
     with pytest.raises(ValueError):
         y = layers.batch_norm(x, '4d')
+
+
+def test_resize_image_like():
+    batch = 1
+    width_height = 5
+    width_height2 = 15
+    channel = 3
+    x = tf.placeholder(tf.float32, [batch, width_height, width_height, channel])
+    s = tf.placeholder(tf.float32, [batch, width_height2, width_height2, channel])
+    x_rshape = images.resize_images_like(x, s)
+    assert x_rshape.get_shape() == s.get_shape()
+
+    sess = tf.Session()
+    sess.run(tf.initialize_all_variables())
+    x_ = np.float32(np.random.rand(batch, width_height, width_height, channel))
+    x_out = sess.run(x_rshape, {x:x_})
+    assert x_out.shape == s.get_shape()
```

Comment:
`s.get_shape()` returns a `TensorShape` where each dimension is of class `Dimension` whereas `x_out.shape` is a list of `int`s. It feels a little weird that they should be comparable. You can get the int from a `Dimension` object using `.value`.